pages/1.11/release-notes/1.11.8/index.md

```diff
@@ -0,0 +1,97 @@
+---
+layout: layout.pug
+navigationTitle:  Release Notes for 1.11.8
+title: Release Notes for 1.11.8
+menuWeight: 5
+excerpt: Release notes for DC/OS 1.11.8
+---
+
+DC/OS 1.11.8 was released on December 6, 2018. 
+
+[button color="purple" href="https://downloads.dcos.io/dcos/stable/1.11.7/dcos_generate_config.sh"]Download DC/OS Open Source[/button]
+
+[button color="light" href="https://support.mesosphere.com/hc/en-us/articles/213198586"]Download DC/OS Enterprise[/button]
+
+DC/OS 1.11.8 includes the following components:
+- Apache Mesos 1.5.x [change log](https://github.com/apache/mesos/blob/2ead30d/CHANGELOG).
+- Marathon 1.6.564 [change log](https://github.com/mesosphere/marathon/tree/3fa693b32).
+- Metronome 0.4.5 [change log](https://github.com/dcos/metronome/releases/tag/v0.4.5).
+
+# Issues fixed in DC/OS 1.11.8
+
+## Docker containers
+- COPS-4087	- For applications that use Docker containers with a Virtual IP address, backend port mapping resolves access to the application by using the `host_IP:port_number` instead of the `container_ip:port_number`.
+
+## GUI
+- COPS-3369 - Updates to the DC/OS UI provide better rendering for elements such as environment variables, secrets, labels, and version information.   
+- DCOS_OSS-1961	- Pod lifecycle stages are supported in the DC/OS UI.
+
+## Marathon
+- COPS-3764	- The upgrade to Marathon 1.6.x enables successful secret validation for secrets included in a Marathon JSON app definition file.
+- MARATHON-8317	- This release adds support for enhancements and issues fixed in Marathon 1.6.x. 
+
+## Mesos
+- COPS-3953	- The Mesos fetcher process automatically retries downloading files using their associated URI if the previously-downloaded and cached versions of the files are not found.
+- DCOS-43544 - Logic changes enable nested containers to run under the same user account as the user associated with their parent container by default. For nested containers in a pod, the default executor’s user--that is, the user running the top-level container--has been the framework user. In a scenario where the framework user is a normal user but the nested container user is `root`, the change in this release enables the second-level nested containers to run as the same user--for example, the `root` user--as the parent top-level container instead of as the framework user by default.
+- DCOS-43593 - This release fixes an issue that could cause Mesos master endpoints—-such as `reserveResources` or `createVolume`-—to fail during authorization. For example, before implementing this fix, the authorization requests for an endpoint might fail or be incomplete if there’s extreme load on the IAM service. The change in this release ensures that authorization requests for an endpint are complete before continuing.
```

Comment:
I would just say "This fixes..." and "This fix ensures" instead of "This release fixes" VS. "The change in this release ensures..."

Reply:
For clarity, it's generally a better practice to specify a subject rather than using an implied subject ("this explicitly-stated noun does xyz" is generally preferred over "this does xyz"). Both forms are acceptable, but constructing the sentence with a subject reduces the chance of ambiguity and makes doc easier to translate into other languages. Unless you feel strongly about the "shorthand" version, I'd prefer to use "this release" over "this" ...